google/resource_compute_address.go

```diff
@@ -74,11 +98,36 @@ func resourceComputeAddressCreate(d *schema.ResourceData, meta interface{}) erro
 	}
 
 	// Build the address parameter
-	addr := &compute.Address{Name: d.Get("name").(string)}
-	op, err := config.clientCompute.Addresses.Insert(
-		project, region, addr).Do()
-	if err != nil {
-		return fmt.Errorf("Error creating address: %s", err)
+	addr := &computeBeta.Address{
+		Name:        d.Get("name").(string),
+		AddressType: d.Get("address_type").(string),
```

Comment:
What is the behavior when `address_type` is empty? Does it assume INTERNAL or EXTERNAL? Can you document it in the `&Schema.schema` block above.